doc/dev/bluestore.rst

```diff
@@ -83,3 +83,60 @@ might be several blobs to consider (e.g., we might be able to *W* into one or
 +--------------------------+--------+--------------+-------------+--------------+---------------+
 | 100 byte clone overwrite | P | N  | P | N        | P | N       | P | N        | N (F?)        |
 +--------------------------+--------+--------------+-------------+--------------+---------------+
+
+
+
+==========
+ Database
+==========
+
+BlueStore uses key-value database to store its state.
+Currently only RocksDB has been integrated.
+
+
+Objects
+-------
+
+Objects and shards are stored in DB under prefix 'O'.
+
+- Key is a serialization of *ghobject_t* oid.
+- Value is a serialization of TODO
```

Comment:
Is `TODO` the intended text here, or is it a placeholder that wasn't circled back to?